### PR TITLE
TFW-3522 - Histogram sort toggle

### PR DESCRIPTION
**Pull request by tVeratti**

### CODE REVIEW

**Comment by tVeratti** [comment]
@troyblank @jasonhale Updated and ready

File: components/visualizations/wordBar.js [hunk 1]
```diff
@@ -65,24 +72,86 @@ export default class WordBar extends PureComponent {
...
     onToggle = ({ value }) => {
         this.setState({ activeValue: value });
     }
 
-    getMenuFromData = data => Object.keys(data).map(key => ({ label: key, value: key }));
+    onSortUpdate = (e) => {
+        const { encoding } = this.props;
+        const { value } = e;
+        let sortBy;
+        if (value.indexOf(ALPHA_KEY) !== -1) {
```
Comment (jasonhale) [inline components/visualizations/wordBar.js]: Can we just use `value.includes(ALPHA_KEY)`...?  I don't remember if IE11 is a requirement.

Comment (troyblank) [reply components/visualizations/wordBar.js]: word to that.

Comment (tVeratti) [reply components/visualizations/wordBar.js]: good call - updated!

**Pull request merged**